freqtrade/plugins/pairlist/VolumePairList.py

```diff
@@ -36,6 +39,35 @@ def __init__(self, exchange, pairlistmanager,
         self._min_value = self._pairlistconfig.get('min_value', 0)
         self._refresh_period = self._pairlistconfig.get('refresh_period', 1800)
         self._pair_cache: TTLCache = TTLCache(maxsize=1, ttl=self._refresh_period)
+        self._lookback_days = self._pairlistconfig.get('lookback_days', 0)
+        self._lookback_timeframe = self._pairlistconfig.get('lookback_timeframe', '1d')
+        self._lookback_period = self._pairlistconfig.get('lookback_period', 0)
+
+        if (self._lookback_days > 0) & (self._lookback_period > 0):
+            raise OperationalException(
+                'Ambigous configuration: lookback_days and lookback_period both set in pairlist '
+                'config. Please set lookback_days only or lookback_period and lookback_timeframe '
+                'and restart the bot.'
+            )
+
+        # overwrite lookback timeframe and days when lookback_days is set
+        if self._lookback_days > 0:
+            self._lookback_timeframe = '1d'
+            self._lookback_period = self._lookback_days
+
+        # get timeframe in minutes and seconds
+        self._tf_in_min = timeframe_to_minutes(self._lookback_timeframe)
+        self._tf_in_sec = self._tf_in_min * 60
+
+        # wether to use range lookback or not
+        self._use_range = (self._tf_in_min > 0) & (self._lookback_period > 0)
+
+        if self._use_range & (self._refresh_period < self._tf_in_sec):
+            raise OperationalException(
+                f'Refresh period of {self._refresh_period} seconds is smaller than one '
+                f'timeframe of {self._lookback_timeframe}. Please adjust refresh_period '
+                f'to at least {self._tf_in_sec} and restart the bot.'
+            )
```

Comment:
You may also wanna top-cap this at the exchange candle limit for one call (look at `AgeFilter` for how to do this).

`refresh_latest_ohlcv` will only return 1 exchange call - so you won't have 2000 5m candles available.

Reply:
Not sure if I understand this correctly, but this is what should happen in line 84ff, where we trigger an `OperationalException` when the `lookback_period` exceedes `ohlcv_candle_limit` for given candlesize?!

Or maybe I misunderstood it.

Reply:
maybe i wrote this comment before getting to line 84 ... :laughing: 

sorry ...